src/databricks/sql/client.py

```diff
@@ -590,6 +596,9 @@ def setoutputsize(self, size, column=None):
         """Does nothing by default"""
         pass
 
+    def setLZ4Compression(self, lz4_compression):
```

Comment:
@moderakh. This will allow to change compression without initialising new cursor or connection. Do we want to give such flexibility?

Reply:
@mohitsingla-db similar to the JDBC/ODBC driver this should be passed to the client upfront, rather than per cursor or per query

something like this, at this layer.

```python
sql.connect(host, httpath, your_new_option)
```

`your_new_option` should be passed as kwargs
and for its naming please check what the equivalent option is named in jdbc/odbc and here name it consistently